cli/cmd/brokers.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(brokersCmd)
+	brokersCmd.AddCommand(createCmd)
+	brokersCmd.AddCommand(describeCmd)
+	brokersCmd.AddCommand(listCmd)
+	brokersCmd.AddCommand(deleteCmd)
+}
+
+var brokersCmd = &cobra.Command{
+	Use:   "brokers",
+	Short: "manage brokers associated with a service controller",
+	Long:  "Manage brokers associated with a service controller",
+}
+
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Add a service broker to service controller",
+	Long:  "Add a service broker to service controller",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return fmt.Errorf("Not implemented yet")
+	},
+}
+
+var describeCmd = &cobra.Command{
+	Use:   "describe",
+	Short: "Describe the specified service broker",
+	Long:  "Describe the specified service broker",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return fmt.Errorf("need name of the broker")
```

Comment:
s/name/ID/
Format is: `GET /v2/service_brokers/6d019023-17e6-488e-a50a-4ce1f35a9524`
https://apidocs.cloudfoundry.org/237/service_brokers/retrieve_a_particular_service_broker.html